src/monodroid/jni/embedded-assemblies.cc

```diff
@@ -381,17 +400,6 @@ EmbeddedAssemblies::gather_bundled_assemblies_from_apk (const char* apk, monodro
 				continue;
 			}
 
-			if (utils.ends_with (cur_entry_name, ".jm")) {
```

Comment:
Won't this break fast deployment?

Reply:
Fast deployment will put the map files in the override directory, not in the apk

Reply:
> Fast deployment will put the map files in the override directory, not in the apk

Not by default actually...

You can opt-into the experimental feature with `AndroidFastDeploymentType=Assemblies:Dexes`.

If a customer creates a File->New project and hits the play button, their typemap files aren't fast deployed. They will be in the APK.

I don't know if anything needs to be changed here, though.